UI/MiscScreens.cpp

```diff
@@ -439,6 +439,12 @@ void LogoScreen::render() {
 
 #ifdef _WIN32
 	dc.DrawText(screenManager()->getThin3DContext()->GetInfoString(T3DInfo::APINAME), bounds.centerX(), bounds.y2() - 100, colorAlpha(0xFFFFFFFF, alphaText), ALIGN_CENTER);
+#elif defined(ANDROID)
```

Comment:
Well, I'd say any ARM64 qualifies as an experimental ARM64 build.  Though not sure what work is required to do iOS.

-[Unknown]